src/pages/projects/ProjectsTabs.tsx

```diff
@@ -6,12 +6,19 @@ import Link from 'next/link'
 
 const TAB_TYPE_NAMES: { [k in ProjectCategory]: string } = {
   all: t`All`,
+  latest: t`Latest`,
   holdings: t`My holdings`,
   myprojects: t`My projects`,
   trending: t`Trending`,
 }
 
-const TABS: ProjectCategory[] = ['all', 'trending', 'holdings', 'myprojects']
+const TABS: ProjectCategory[] = [
+  'all',
+  'latest',
```

Comment:
I think we should switch the order of `Trending` and `Latest`. I think `Trending` should take precedence over latest 👍

Reply:
I'm not sure about this, but it seems great to me any way!

The only thing i placed before trending was that maybe 2 infinite query tabs should appear one after the other, but i couldn't reveal any clear UX logic that should apply here.

but like i indicated, i have no clear reason in this scenario. I'll place it after trending!